src/redux/snackbar/snackbar.actions.js

```diff
@@ -0,0 +1,22 @@
+import {
+  SET_SNACKBAR_MESSAGE,
+  SET_SNACKBAR_SEVERITY,
+  SET_SNACKBAR_STATUS
+} from './snackbar.types';
+
+const setSnackBarStatus = (newSnackBarStatus) => ({
+  type: SET_SNACKBAR_STATUS,
```

Comment:
Do we really need to have three separate actions for setting the snackbar? As I understand we don't update the message inside it, but just display a new one and close old. If it is true it is better to have only two actions: `showSnackbar` and `closeSnackbar`

Reply:
We update also type(error or success) and message of snackbar, not only status, so i guess it's better to have separate actions for all this stuff